### PR TITLE
Documentation/upgrades: add instruction to stop an etcd process

### PR DESCRIPTION
**Pull request by radhikapc**
Updated the upgrade documents to reflect instruction to stop etcd process before upgrading

### CODE REVIEW

File: Documentation/upgrades/upgrade_3_0.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ $ curl http://localhost:2379/version
 
 #### 2. Stop the existing etcd process
 
+Stop the existing etcd process:
+
+```
```
Comment (heyitsanthony) [inline Documentation/upgrades/upgrade_3_0.md]: \```sh

File: Documentation/upgrades/upgrade_3_0.md [hunk 2]
```diff
@@ -57,6 +57,12 @@ $ curl http://localhost:2379/version
 
 #### 2. Stop the existing etcd process
 
+Stop the existing etcd process:
```
Comment (heyitsanthony) [inline Documentation/upgrades/upgrade_3_0.md]: If running etcd with systemd, stop each running etcd process with `systemctl`:

File: Documentation/upgrades/upgrade_3_1.md [hunk 3]
```diff
@@ -58,6 +58,12 @@ $ curl http://localhost:2379/version
 
 #### 2. Stop the existing etcd process
 
+Stop the existing etcd process:
+
+```
```
Comment (heyitsanthony) [inline Documentation/upgrades/upgrade_3_1.md]: \```sh

File: Documentation/upgrades/upgrade_3_1.md [hunk 4]
```diff
@@ -58,6 +58,12 @@ $ curl http://localhost:2379/version
 
 #### 2. Stop the existing etcd process
 
+Stop the existing etcd process:
```
Comment (heyitsanthony) [inline Documentation/upgrades/upgrade_3_1.md]: If running etcd with systemd, stop each running etcd process with `systemctl`:

**Comment by radhikapc** [comment]
adding reviewers  @heyitsanthony  @gyuho

**Comment by gyuho** [comment]
@joshix @xiang90 

1. systemd specific commands do not belong to upgrade doc, 
2. If we do, it should be separate documentation
3. We had `systemctl`-specific introductions before this commit https://github.com/coreos/docs/commit/9bdc9ddd9b52fd7700617818d8b72cfd758bee88 but it's gone now.

I don't see anywhere to put now.

I have full systemd instructions here http://play.etcd.io/install.
Should we have separate instructions here https://github.com/coreos/etcd/tree/master/Documentation/platforms?

**Comment by radhikapc** [comment]
I was following the documentation to bring up container linux + etcd on my laptop, and found that in the given section there is no instruction on how to stop an etcd process, and tried this method and worked. 
cc @crawford

**Comment by gyuho** [comment]
@radhikapc Yeah etcd doesn't have good instructions around systemd now :(
We try to have clear distinction between docs, like we have separate one for running etcd on containers.

**Pull request closed without merging**